products/developments/housing_by_cb/metadata.yml

```diff
@@ -0,0 +1,219 @@
+id: housing_database_by_2020_census_block
+
+attributes:
+  description: |-
+    Net change in housing units arising from new buildings, demolitions, or alterations for NYC Census Blocks since 2010.  The NYC Department of City Planning's (DCP) <a href="https://www1.nyc.gov/site/planning/data-maps/open-data/dwn-housing-database.page#housingdevelopmentproject">Housing Database</a> provide the 2010 census count of housing units, the net change in Class A housing units since the census, and the count of units pending completion for commonly used political and statistical boundaries. These tables are aggregated from the DCP Housing Database, which is derived from Department of Buildings (DOB)-approved housing construction and demolition jobs filed or completed in NYC since January 1, 2010. Net housing unit change is calculated as the sum of all three construction job types that add or remove residential units: new buildings, major alterations, and demolitions, and can be used to determine the change in legal housing units across time and space.
+    All previously released versions of this data are available at a <a href="https://www1.nyc.gov/site/planning/data-maps/open-data/bytes-archive.page?sorts[year]=0"BYTES of the BIG APPLE- Archive</a>
+  display_name: Housing Database by 2020 Census Block
+  each_row_is_a: census block
+  tags:
+  - housing
+  - house
+  - building
+  - development
+  - dwelling unit
+  - certificate of occupancy
+  - co
+  - permit
+  - job application
+  - filing
+  - net units
+  - unit change
+  - pipeline
+  - construction
+  - census block
+  - census tract
+  - nta
+  - puma
+  - city council district
+  - community district
+
+assembly: []
+
+custom: {}
+
+destinations:
+- id: socrata_prod
+  type: socrata
+  files:
+  - id: primary_shapefile
+    custom:
+      destination_use: dataset_file
+  custom:
+    four_four: 8zyi-kxay
+
+files:
+- file:
+    id: primary_shapefile
+    filename: HousingDB_by_2020_CensusBlock.shp.zip
+    type: shapefile
+- file:
+    id: Housing Summary Files - ReadMe.pdf
+    filename: Housing Summary Files - ReadMe.pdf
+    is_metadata: true
+    type: None
+- file:
+    id: Housing_Summary_Files_Data_Dictionary.xlsx
+    filename: Housing_Summary_Files_Data_Dictionary.xlsx
+    is_metadata: true
+    type: None
+
+columns:
+- id: the_geom
+  name: the_geom
+  data_type: geometry
+  description: geometry of the Census Block
+  example: None
+  custom:
+    readme_data_type: geometry
+- id: cenblock2020
+  name: cenblock2020
+  data_type: text
+  example: '360470569002003'
+- id: bctcb2020
+  name: bctcb2020
+  data_type: text
+  example: '41241001018'
+- id: comp2010ap
+  name: comp2010ap
+  data_type: integer
+  description: |
+    Net number of class A housing units completed between April 1, 2010
```

Comment:
@alexrichey , curious on what norms you've settled on - do we just prefer longer one liners? Wrap things like this? Or other norms similar to that

Reply:
Good question. I've been inconsistent with it myself. Two thoughts:
- might be easiest to match whatever comes out of the yaml serializer
- would be ideal if folks with less yaml editing experience could read and edit, which is tough when the line is too long. 
Curious what you think though